R/pkg/inst/tests/testthat/test_sparkSQL.R

```diff
@@ -371,13 +371,13 @@ test_that("Collect DataFrame with complex types", {
   expect_equal(bob$height, 176.5)
 })
 
-test_that("read.json()/jsonFile() on a local file returns a DataFrame", {
+test_that("read.json()/jsonFile() and write.json()", {
   df <- read.json(sqlContext, jsonPath)
   expect_is(df, "DataFrame")
   expect_equal(count(df), 3)
   # read.json()/jsonFile() works with multiple input paths
   jsonPath2 <- tempfile(pattern="jsonPath2", fileext=".json")
-  write.df(df, jsonPath2, "json", mode="overwrite")
+  write.json(df, jsonPath2)
```

Comment:
I think it actually make sense to have a test for `write.df` with additional option (in this case `mode="overwrite"`) - is there a reason for removing this?